ci/Jenkinsfile.linux

```diff
@@ -13,6 +13,8 @@ pipeline {
     LANG = 'en_US.UTF-8'
     LANGUAGE = 'en_US.UTF-8'
     LC_ALL = 'en_US.UTF-8'
+    QT_PATH = "/opt/qt59"
+    PATH = "/opt/qt59/bin:${env.PATH}"
```

Comment:
This will DEFINITELY not work. Rebase from devel, you'll see the comment:
https://github.com/status-im/status-react/blob/develop/ci/Jenkinsfile.linux#L26